.github/workflows/ci.yaml

```
@@ -0,0 +1,45 @@
+---
+
+name: CI
+
+on:
+  pull_request:
+    branches: [devel]
+
+  push:
+    branches: [devel]
+
+jobs:
+  pull_request:
+    runs-on: ubuntu-18.04
+    name: pull_request
+    env:
+      DOCKER_API_VERSION: "1.38"
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"
+
+      - name: Install Dependencies
+        run: |
+          pip install \
+            "molecule<3.5" \
+            molecule-docker \
+            yamllint \
+            ansible-lint \
+            openshift \
+            jmespath \
+            ansible-core
+      - name: Install Collections
+        run: |
+         ansible-galaxy collection install community.general kubernetes.core:2.3.2 operator_sdk.util
```

I guess the `community.general` collection was added for the `docker_image` module but it's not present in the general collection since 2.0.0 so you'll need to use the community.docker collection instead
```suggestion
         ansible-galaxy collection install -r requirements.yml
         ansible-galaxy collection install community.docker
```